packages/runner/test/module.test.ts

```diff
@@ -243,4 +243,53 @@ describe("module", () => {
       expect([...nodes][0].inputs.$event).toBe(stream);
     });
   });
+
+  describe("action function", () => {
+    it("creates a handler factory", () => {
+      const increment = action<{ count: number }>(({ count }) => {
```

Comment:
this won't test the transformed version (and as mentioned in the comment above, this isn't particularly useful outside of CTS), i'd just remove this test / change to one that checks for this throwing, if you go down that route.